protobuf_test.go

```diff
@@ -217,3 +216,23 @@ func TestTimeTypesEncodeDecode(t *testing.T) {
 	assert.Equal(t, in.Time.UnixNano(), out.Time.UnixNano())
 	assert.Equal(t, in.Duration, out.Duration)
 }
+
+type CipherText struct {
+	A, B int
```

Comment:
This would translate to in vanilla protobuf (or proto2)
```
message CypherText {
  required int32 a = 1;
  required int32 b = 2;
}
```
Hence, https://github.com/dedis/protobuf/pull/17/commits/d88c9f984b233c505a6350b32f8c1f75db01c1b3#diff-6e7837ff180a7fb755c13cd19e95dbd5R232
would fail encoding with the error `required field "A" not set`. If we want our protobuf implementation compatible to orig. protobuf our library should behave the same this shouldn't even encode if the struct is defined as above (hasn't anything to do with maps though).
If changed to `A, B *int` and this would be optional and this would (almost) be encodable using protobuf.